Add IoT class to Group

jordibsk10 opened the pull request:
As requested in #14661 this is to add the IoT class to Group.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

jordibsk10 committed:
Add IoT class to Group

frenck commented on source/_integrations/group.markdown:
```diff
@@ -4,6 +4,7 @@ description: Instructions on how to setup groups within Home Assistant.
 ha_category:
   - Organization
 ha_release: pre 0.7
+ha_iot_class: Local Push
```
A group's state is determined by its members, so I think calculated would be more fitting here.

```suggestion
ha_iot_class: Calculated
```

jordibsk10 committed:
Update source/_integrations/group.markdown

Yes, we can say that group' state gets calculated based on the state of its members.
It doesn't have a direct communication with devices, so you are right that Local Push may not fit 100% here.
Thanks @frenck

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck reviewed (approved):
Thanks, @jordibsk10 👍 

Happy Hacktoberfest 🎉
